src/Microsoft.AspNetCore.Mvc.DataAnnotations/Internal/FileExtensionsAttributeAdapter.cs

```diff
@@ -3,16 +3,28 @@
 
 using System;
 using System.ComponentModel.DataAnnotations;
+using System.Linq;
 using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
 using Microsoft.Extensions.Localization;
 
 namespace Microsoft.AspNetCore.Mvc.DataAnnotations.Internal
 {
     public class FileExtensionsAttributeAdapter : AttributeAdapterBase<FileExtensionsAttribute>
     {
+        private readonly string _extensions;
+        private readonly string _formattedExtensions;
+
         public FileExtensionsAttributeAdapter(FileExtensionsAttribute attribute, IStringLocalizer stringLocalizer)
             : base(attribute, stringLocalizer)
         {
+            // Build the extension list based on how the JQuery Validation's 'extension' method expects it
+            // https://jqueryvalidation.org/extension-method/
+
+            // These lines follow the approach what the FileExtensionsAttribute does internally
```

Comment:
Think you mean `These lines follow the same approach as the FileExtensionsAttribute.`